src/privatekey.cpp

```diff
@@ -160,6 +172,11 @@ const G2Element& PrivateKey::GetG2Element() const
     return g2Cache;
 }
 
+bool PrivateKey::HasKeyData() const
```

Comment:
Seems to be a functional equivalent to `HasKeyData`, can we just call that instead?

Reply:
Not exactly, one throws an exemption.